File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 03/Lab3C#/AthleticsCompetition/domain/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AthleticsCompetition.domain
{
    class Person
    {
        /// <summary>
        /// getter and setter for person ID
        /// </summary>
        protected int Id { get; set; }

        /// <summary>
        /// getter and setter for person first name
        /// </summary>
        protected string FirstName { get; set; }

        /// <summary>
        /// getter and getter for person last name
        /// </summary>
        protected string LastName { get; set; }

        /// <summary>
        /// constructor for preson
        /// </summary>
        /// <param name="firstName"> string </param>
        /// <param name="lastName"> string </param>
        public Person(string firstName, string lastName)
        {
            this.FirstName = firstName;
            this.LastName = lastName;
        }

    }
}

File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 03/Lab3JAVA/AthleticsCompetition/src/domain/Person.java
package domain;

public class Person {
    Integer id;
    String firstName;
    String lastName;


    /**
     * constructor for perosn
     *
     * @param firstName - string
     * @param lastName  - string
     */
    public Person(String firstName, String lastName) {
        this.firstName = firstName;
        this.lastName = lastName;
    }

    /**
     * getter methode for id
     *
     * @return - integer , id of the child
     */
    public Integer getId() {
        return id;
    }

    /**
     * getter methode for firstName
     *
     * @return - string, first name of the child
     */
    public String getFirstName() {
        return firstName;
    }

    /**
     * setter methode for firstName
     *
     * @param firstName - string, represents the first name of the child
     */
    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    /**
     * getter methode for lastName
     *
     * @return - string, represents the last name of the child
     */
    public String getLastName() {
        return lastName;
    }

    /**
     * setter methode for lastName
     *
     * @param lastName -  string, represents the last name of the chila
     */
    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

}
